### PR TITLE
BNGP-1270 Multiple journey refactoring

### PR DESCRIPTION
**Pull request by pwadmore-ea**
https://eaflood.atlassian.net/browse/BNGP-1270

Refactor to support the landowner and developer journeys

### CODE REVIEW

File: packages/azure-functions/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Node.js Microsoft Azure functions associated with procesing an application to the BNG public register.
 Microsoft Azure SignalR is used to provide asynchronous notifications associated with function processing to the
-[application to register web application](../application-to-register-webapp/).
+[application to register web application](../webapp/).
```
Comment (teddmason) [inline packages/azure-functions/README.md:5]: do you want to update this one?

File: packages/webapp/src/utils/constants.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ const NO = 'no'
 const NO_AGAIN = 'noAgain'
 const OS_API_TOKEN = 'land/os-api-token'
 const PUBLIC = 'public'
+const PUBLIC_ROUTES = `${PUBLIC}-routes`
```
Comment (teddmason) [inline packages/webapp/src/utils/constants.js:55]: Is it necessary to have a PUBLIC constant here? it's not used anywhere else now that PUBLIC_ROUTES is added to routes array.

File: packages/webapp/src/utils/upload.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import path from 'path'
 import { uploadDocument } from '@defra/bng-document-service'
 import multiparty from 'multiparty'
-import constants from '../utils/constants.js'
+import constants from './constants.js'
```
Comment (teddmason) [inline packages/webapp/src/utils/upload.js:4]: ha good spot, how did you even notice?

**Review by teddmason — CHANGES_REQUESTED** [review]

File: packages/azure-functions/README.md (hunk 1 above)
Comment (pwadmore-ea) [reply packages/azure-functions/README.md]: Done

File: packages/webapp/src/utils/constants.js (hunk 2 above)
Comment (pwadmore-ea) [reply packages/webapp/src/utils/constants.js]: PUBLIC constant removed as suggested.

File: packages/webapp/src/utils/upload.js (hunk 3 above)
Comment (pwadmore-ea) [reply packages/webapp/src/utils/upload.js:4]: Thanks - somehow I noticed

**Review by teddmason — APPROVED** [review]

**Pull request merged**